core/src/main/java/me/lokka30/treasury/plugin/core/utils/UpdateChecker.java

```diff
@@ -43,21 +46,75 @@ public static void checkForUpdates() {
         if (!plugin.configAdapter().getSettings().checkForUpdates()) {
             return;
         }
```

Comment:
it doesn't matter at all where this check would be done in the method or before it getting called. I put it here just in case somehow there is a method to support the SRP (Single Responsibility Principle) in any of the platforms we plan to support.

Reply:
I disagree - I think it makes the code cleaner. Otherwise anywhere the update check method is called, we will also have to include that check.

Reply:
@MrNemo64 is this conversation resolved?

Reply:
Yes